shared/src/business/useCases/sealCaseInteractor.test.js

```diff
@@ -8,6 +8,9 @@ describe('sealCaseInteractor', () => {
     applicationContext
       .getPersistenceGateway()
       .getCaseByDocketNumber.mockReturnValue(MOCK_CASE);
+
+    applicationContext.getNotificationGateway().sendNotificationOfSealing =
```

Comment:
Nit: can this be removed? It looks like `sendNotificationOfSealing` is retrieved from `getDispatchers` which is already mocked in `createTestApplicationContext`